src/controller/java/src/matter/onboardingpayload/OnboardingPayload.kt

```diff
@@ -415,7 +405,7 @@ class OnboardingPayload(
    *
    * @param info Optional QR code info object to add
    */
-  private fun addOptionalVendorData(info: OptionalQRCodeInfo) {
+  fun addOptionalVendorData(info: OptionalQRCodeInfo) {
```

Comment:
This in internal implementation and should be called from fun addOptionalVendorData(tag: Int, data: String), why remove the private accessor?

Reply:
Remove Made this method private again and we are now calling the public addOptionalVendorData(tag, data) method.

Reply:
thanks